package.json

```diff
@@ -60,90 +49,73 @@
     ]
   },
   "dependencies": {
-    "@uirouter/angularjs": "1.0.15",
+    "@uirouter/angularjs": "1.0.18",
     "angular": "1.7.0",
     "angular-animate": "1.7.0",
     "angular-aria": "1.7.0",
     "angular-local-storage": "0.7.1",
-    "angular-localforage": "1.3.7",
-    "angular-material": "1.1.7",
+    "angular-localforage": "1.3.8",
+    "angular-material": "1.1.9",
     "angular-messages": "1.7.0",
     "angular-qrcode": "7.2.0",
     "angular-sanitize": "1.7.0",
     "angular-zxcvbn": "3.2.2",
-    "async": "2.6.0",
-    "babel-runtime": "^6.26.0",
-    "bignumber.js": "4.1.0",
+    "async": "2.6.1",
+    "bignumber.js": "7.2.1",
     "bluebird": "3.5.1",
     "crypto": "1.0.1",
     "decompress": "4.2.0",
-    "electron-compile": "6.4.2",
-    "electron-log": "2.2.14",
-    "electron-settings": "3.1.4",
-    "electron-squirrel-startup": "1.0.0",
-    "electron-updater": "^2.18.2",
-    "ethereumjs-tx": "1.3.3",
-    "ethereumjs-util": "5.1.2",
-    "fibers": "2.0.0",
-    "fs-extra": "5.0.0",
-    "google-charts": "1.0.0",
+    "ethereumjs-tx": "1.3.4",
+    "ethereumjs-util": "5.2.0",
+    "fibers": "2.0.2",
+    "fs-extra": "6.0.1",
+    "google-charts": "1.1.0",
     "is-online": "^7.0.0",
-    "keccak": "1.2.0",
-    "keythereum": "1.0.2",
-    "knex": "0.14.4",
-    "ledgerco": "1.2.1",
-    "mime-types": "2.1.17",
+    "keccak": "1.4.0",
+    "keythereum": "1.0.4",
+    "knex": "^0.14.6",
+    "ledger-wallet-provider": "2.0.1-beta.2",
+    "mime-types": "2.1.18",
     "mv": "2.1.1",
-    "promise-timeout": "^1.3.0",
-    "qrcode-generator": "1.3.1",
-    "request": "2.83.0",
-    "secp256k1": "3.2.5",
-    "sjcl": "1.0.6",
-    "sqlite3": "3.1.13",
-    "uuid": "3.0.0",
-    "web3": "1.0.0-beta.33",
+    "qrcode-generator": "1.4.0",
+    "request": "2.87.0",
+    "secp256k1": "3.5.0",
+    "sjcl": "1.0.7",
+    "source-map-support": "^0.5.6",
+    "sqlite3": "^4.0.0",
+    "uuid": "3.2.1",
+    "web3": "1.0.0-beta.34",
+    "web3-provider-engine": "14.0.5",
+    "webpack-sources": "1.1.0",
     "zxcvbn": "4.4.2"
   },
   "devDependencies": {
-    "babel-plugin-transform-async-to-generator": "6.24.1",
-    "babel-preset-env": "1.6.1",
-    "babel-preset-es2015": "6.24.1",
     "chai": "^4.1.2",
-    "chalk": "^2.4.1",
-    "commitizen": "^2.10.1",
-    "cz-conventional-changelog": "^2.1.0",
-    "delay": "^2.0.0",
+    "chalk": "2.3.2",
+    "copy-webpack-plugin": "^4.5.1",
+    "delay": "^3.0.0",
+    "electron": "^2.0.2",
```

Comment:
I don't think this is a good idea - should stick to electron 1.8.4 as required via other dependency - cc: @dfcowell